config.py

```diff
@@ -141,6 +30,22 @@ def __init__(self):
         if not os.path.exists(self.captionsFolder):
             os.makedirs(self.captionsFolder)
 
+        self.videoToUse = ''
+
+        self.saveFolder = "savedData"
+        self.fileTypes = ["topicModel", "topicsOverTime"]
+
+        for folder in self.fileTypes:
+            folderPath = os.path.join(self.saveFolder, folder)
+            if not os.path.exists(folderPath):
+                os.makedirs(folderPath)
```

Comment:
Again, some basic error checking for file/folder doesn't exist error

Reply:
Isn't the error checking being handled by making the folders if they do not exist? My thought here was to just make the folders if they do not exist, to avoid any trouble later on with the data loader and saver.

Reply:
Yes, you are correct. `os.path.exists(folderPath)` is the helper function that is doing that check for you Consider this resolved